cinema/permissions.py

```diff
@@ -0,0 +1,12 @@
+from rest_framework.permissions import BasePermission, SAFE_METHODS
+
+
+class IsAdminOrIfAuthenticatedReadOnly(BasePermission):
+    def has_permission(self, request, view):
+        return bool(
+            (
+                request.method in SAFE_METHODS
+                and request.user.is_authenticated
+            )
```

Comment:
"in" operator returns boolean, is_authenticated as well, so what is the purpose to convert it to bool?